CMakeLists.txt

```diff
@@ -14,36 +14,48 @@ include(GNUInstallDirs)
 
 set(CMAKE_CXX_STANDARD 11)
 
+set(ALE_BUILD_LOAD ON CACHE BOOL "If the C++ Python load interface should be built.")
+
 # Third Party Dependencies
 find_package(Eigen3 3.3    REQUIRED NO_MODULE)
 find_package(nlohmann_json REQUIRED)
 
-# If there is an Anaconda environment activated, search that for Python first
-if(EXISTS $ENV{CONDA_PREFIX})
-  message("Searching $ENV{CONDA_PREFIX} for Python libraries")
-  set(Python_ROOT_DIR $ENV{CONDA_PREFIX})
-  set(Python_FIND_STRATEGY LOCATION)
+if(ALE_BUILD_LOAD)
+  # If there is an Anaconda environment activated, search that for Python first
+  if(EXISTS $ENV{CONDA_PREFIX})
+    message("Searching $ENV{CONDA_PREFIX} for Python libraries")
+    set(Python_ROOT_DIR $ENV{CONDA_PREFIX})
+    set(Python_FIND_STRATEGY LOCATION)
+  endif()
+  find_package(Python REQUIRED COMPONENTS Development)
```

Comment:
The problem is whether you add the Python library as a dependency. Cmake won't even let you compile if it can't find the Python library and you add it as a dependency.